media/EdgetpuCfgEditor/index.js

```diff
@@ -120,267 +66,45 @@ function main() {
-  
-  updateCompilerUI();
-
-  compilerSelector.addEventListener("click", function(){
-    updateCompiler();
-    updateCompilerUI(); 
-    applyUpdates();
-  });
-}
-
-function registerSteps() {
-  const checkboxImport = document.getElementById("checkboxImport");
-  const checkboxOptimize = document.getElementById("checkboxOptimize");
-  const checkboxQuantize = document.getElementById("checkboxQuantize");
-  const checkboxCodegen = document.getElementById("checkboxCodegen");
-  const checkboxProfile = document.getElementById("checkboxProfile");
-  const stepImport = document.getElementById("stepImport");
-  const stepOptimize = document.getElementById("stepOptimize");
-  const stepQuantize = document.getElementById("stepQuantize");
-  const stepCodegen = document.getElementById("stepCodegen");
-  const stepProfile = document.getElementById("stepProfile");
-
-  checkboxImport.addEventListener("click", function () {
-    updateSteps();
-    updateImportInputModelType();
-    applyUpdates();
-  });
-  checkboxOptimize.addEventListener("click", function () {
-    updateSteps();
-    updateOptimize();
-    applyUpdates();
-  });
-  checkboxQuantize.addEventListener("click", function () {
-    updateSteps();
-    updateQuantizeActionType();
-    applyUpdates();
-  });
-  checkboxCodegen.addEventListener("click", function () {
-    updateSteps();
-    updateCodegen();
-    applyUpdates();
-  });
-  checkboxProfile.addEventListener("click", function () {
-    updateSteps();
-    updateProfile();
-    applyUpdates();
-  });
-
-  stepImport.addEventListener("click", function () {
-    updateStepUI("Import");
-  });
-  stepOptimize.addEventListener("click", function () {
-    updateStepUI("Optimize");
-  });
-  stepQuantize.addEventListener("click", function () {
-    updateStepUI("Quantize");
-  });
-  stepCodegen.addEventListener("click", function () {
-    updateStepUI("Codegen");
-  });
-  stepProfile.addEventListener("click", function () {
-    updateStepUI("Profile");
-  });
-}
-
 function registerCompilerStep() {
   const checkboxEdgeTPUCompile = document.getElementById("checkboxEdgeTPUCompile");
   const checkboxEdgeTPUProfile = document.getElementById("checkboxEdgeTPUProfile");
   const stepEdgeTPUCompile = document.getElementById("stepEdgeTPUCompile");
   const stepEdgeTPUProfile = document.getElementById("stepEdgeTPUProfile");
 
   checkboxEdgeTPUCompile.addEventListener("click", function () {
+    console.log("compile event clicked");
     updateEdgeTPUStep();    
     applyUpdates();
   });
   checkboxEdgeTPUProfile.addEventListener("click", function () {
+    console.log("profile event clicked");
     updateEdgeTPUStep();    
     applyUpdates();
   });
- 
+  
   stepEdgeTPUCompile.addEventListener("click", function () {
+    console.log("compile step event clicked");
     updateStepUI("EdgeTPUCompile");
   });
   stepEdgeTPUProfile.addEventListener("click", function () {
+    console.log("profile step event clicked");
```

Comment:
I think we should remove this logging code in this PR or later.

Reply:
Thanx, I removed all and committed. Plz check it.